gmm_client/src/client.erl

```diff
@@ -0,0 +1,105 @@
+%%%-------------------------------------------------------------------
+%%% @author Svatopluk
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 10. maj 2021 16:54
+%%%-------------------------------------------------------------------
+-module(client).
+-author("Svatopluk").
+
+
+%% API
+-export([get_id_list/0, get_id_number/0, get_user_info/1, delete_user/1, add_user/1, test/0]).
+
+test()->
+  ok.
+
+url_address() ->
+  "localhost:8080/".
```

Comment:
Jeżeli to adres serwera do którego wykonujemy zapytania, to to nie powinna być stała, zarówno host jak i port mogą się zmienić. Zerknij też na moduł [uri_string](https://erlang.org/doc/man/uri_string.html)

Stałe w Erlangu raczej definiujemy jako makra:
```erlang
-define(URL, "localhost:8080").
```